rotation.go

```diff
@@ -299,8 +303,10 @@ type setStaticAccountOutput struct {
 // tasks must be handled outside of this method.
 func (b *backend) setStaticAccountPassword(ctx context.Context, s logical.Storage, input *setStaticAccountInput) (*setStaticAccountOutput, error) {
 	var merr error
+	// Re-use WAL ID if present, otherwise PUT a new WAL
+	output := &setStaticAccountOutput{WALID: input.WALID}
 	if input == nil || input.Role == nil || input.RoleName == "" {
```

Comment:
Should the `input == nil` check be before the `input.WALID` reference above? Probably will need to split the conditional if so and we still want to return `output` in the error case.

Reply:
Good point. Technically, with the way the response from this function is handled, we only _need_ to return the WAL ID if it changes during this function, but I was moving it up to the top just to be defensive. I've moved it down a little.